### ISSUE TITLE
Slow loading when opening the dashboard.

### ISSUE THREAD

**Issue by WarC0zes** [Clooos/Bubble-Card#108]
Hi,
I noticed the dashboard loading slowly when I access it. I see all the popups cards appear on the screen and disappear.
![bubble4](https://github.com/Clooos/Bubble-Card/assets/91471397/f414bb78-74d8-4a87-bcd6-204b06704cb1)

I'm using v1.2.0.
Need additional information?

**Comment by Clooos** [comment]
Hi and thanks for your feedback! Can you try to install the previous version to see if there is the same behavior? And also can you send me a record of what the dashboard loading looks on this version?

**Comment by WarC0zes** [comment]
Hi,
with version 1.2.3, it's good on mobile but still the same on PC.
Maybe my setup with 10 horizontal-buttons and 10 popups are too heavy?

my config pc:
win10
i7 9700
16Go DDR4
Chrome navigator (last version).

**Comment by Clooos** [comment]
Have you tried to clear your PC browser cache?

**Comment by WarC0zes** [comment]
> Have you tried to clear your PC browser cache? Because your config should be way faster than your mobile.

Yes, cache clear.
![bubble6](https://github.com/Clooos/Bubble-Card/assets/91471397/13f1f583-f058-4f7f-8b3e-caeecb3801e0)

on mobile, I see 1 column loading, then the screen becomes empty with the clock.
on pc, I see all the columns loading, then the screen goes blank with the clock.
between version 1.2.0 and 1.2.3, performance level.

**Comment by Clooos** [comment]
Can you tell me what was the last version that was working for you without this behavior? And can you try to reinstall it to see if it works like it should?

**Comment by WarC0zes** [comment]
1.1.2 is OK.
![bubble7](https://github.com/Clooos/Bubble-Card/assets/91471397/79f71728-0802-46f8-a808-048d4d134619)

**Comment by WarC0zes** [comment]
Hi,
1.2.4 same problem.
I tried with chrome and edge, I cleared the cache. I don't see any errors in the Chrome console.

**Comment by Clooos** [comment]
Can you tell me if it's working in v1.3.0?

**Comment by WarC0zes** [comment]
no, same as the version 1.2.4.

**Comment by WarC0zes** [comment]
same as the version 1.3.3 on desktop with chrome and edge.
![bubble9](https://github.com/Clooos/Bubble-Card/assets/91471397/fb393397-a428-4a62-8635-d54f1a886112)

**Comment by WarC0zes** [comment]
Hi,
same as the version 1.4.2 on desktop with chrome and edge.
On mobile, it's better but I can see columns loading sometimes.

**Comment by Clooos** [comment]
Hi! Here is a new beta version, can you please try to follow the instructions to configure it? This should totally fix your issue 🙂 

https://github.com/Clooos/Bubble-Card/releases/edit/v1.5.0-beta.1

**Comment by WarC0zes** [comment]
Hi,
Sorry but it's still the same. I followed your indication for the beta, clear the cache...
![bubble11](https://github.com/Clooos/Bubble-Card/assets/91471397/1bca0e03-d58b-4dce-a99b-9e2bd3f8904d)

**Comment by Clooos** [comment]
I see that you have an error with card-mod, I guess that you've added it as a frontend module too (I had this error too). Just remove card-mod from resources and you should not have this error anymore, this is possible that this is why you have still this issue.

**Comment by WarC0zes** [comment]
Unfortunately it's not a card-mod problem.
![bubble14](https://github.com/Clooos/Bubble-Card/assets/91471397/a046cf93-3ef1-4d7f-a09e-4bf4832f5f14)

I have an error message in HACS, as I deleted the resources. Can I ignore it?
![bubble error hacs](https://github.com/Clooos/Bubble-Card/assets/91471397/21dda962-5a6a-4a04-9bb9-13e0f80f9560)

**Comment by Clooos** [comment]
Yes I have this too, and I also have the card-mod error (but only once) like you now, are you sure that you don't have any other error in your console?

**Comment by WarC0zes** [comment]
No, the 2 errors you see, its from apexcharts-card.
Error: <g> attribute transform: Expected number, "translate(NaN, 0)".

**Comment by Clooos** [comment]
This should be THE one 🤞
https://github.com/Clooos/Bubble-Card/releases/tag/v1.5.0-beta.2

**Comment by WarC0zes** [comment]
Hi,
![bubble15](https://github.com/Clooos/Bubble-Card/assets/91471397/a361b30f-b62f-43ca-934d-b252b343347c)
There is an error in the chrome console.
![bubble error console beta2](https://github.com/Clooos/Bubble-Card/assets/91471397/04471871-54c5-4597-9f3a-a5b1de87f6f7)

I may have misunderstood the readme, but the bubble-pop-up card is not added to the bubble-card folder. I downloaded the source code and manually copied the bubble-pop-up.js file.

**Comment by Clooos** [comment]
Bubble Card should not be added as an extra module in this version, this is why you have this error.

**Comment by WarC0zes** [comment]
Ok, I correct my config and test

**Comment by WarC0zes** [comment]
Config OK, no errors in console.
But that doesn't change anything, it's still slow.
I cleared the cache of chrome.

**Comment by WarC0zes** [comment]
Would another card be the problem?
![card use](https://github.com/Clooos/Bubble-Card/assets/91471397/27b05593-0521-467f-8cca-fd0451619d83)
![card use2](https://github.com/Clooos/Bubble-Card/assets/91471397/59c08a90-4578-428b-b519-20bf860da006)

**Comment by Clooos** [comment]
I have only 7 custom cards in my setup, and you have a lot of them, it's most likely because they all have to be loaded before the pop-ups initialization. I still don't understand why it's working for you in 1.0.3.

But I will soon try to add something to hide the dashboard during the pop-ups initialization, I will tell you when it will be available! 🤞

**Comment by WarC0zes** [comment]
no change with 1.5.3, shame.

**Comment by Clooos** [comment]
Have you tried with a new dashboard to see what custom card is causing this? Because your issue looks like some kind of conflict with a custom card.

And also are you in Optimized mode?

**Comment by WarC0zes** [comment]
> Have you tried with a new dashboard to see what custom card is causing this? Because your issue looks like some kind of conflict with a custom card.

No, I didn't try. I will try.



> And also are you in Optimized mode?

Yes.
I have the popup card, in the card selection.
